ci/validate-vendor.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Go Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+#
+# This script checks if we changed anything with regard to dependency management
+# for our repo and makes sure that it was done in a valid way.
+#
+# This file is a slightly modified copy of https://github.com/golang/dep/blob/master/hack/validate-vendor.bash
+# The only change made was checking if `dep` was available and installing it if necessary.
+
+
+set -e -o pipefail
+
+if [ -z "$VALIDATE_UPSTREAM" ]; then
+	VALIDATE_REPO='https://github.com/status-im/status-go'
+	VALIDATE_BRANCH='develop'
```

Comment:
Shouldn't branch be taken from env and only default to `develop`? We merge almost every branch to `develop` but it should change soon.

Reply:
👍